### ISSUE TITLE
(macOS) Metadata - Dropdown for 'Genre' tag is inaccessible via keyboard

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#4958]
### Bug description

If the User attempts to change the 'Genre' metadata tag, the drop-down constantly disappears, and doesn't allow for streamlined use of the drop-down IF the User tries to browse the list via keyboard shortcuts (Up and Down arrows)

### Steps to reproduce

1. Launch Audacity
2. Import or generate audio on a stereo or mono track
3. Go to File -> Export Audio (or press Shift+Ctrl(CMD)+E)
4. Click 'Metadata...'
5. Double-click to the 'Genre' value to focus text input
6. Press 'Down' arrow
7. Observe result

### Expected behavior

Upon pressing the 'Down' key, the list of genres should be available to the User to select and scrollable with 'Up' and 'Down' keys.

### Actual behavior

The list cannot be scrolled with 'Up' and 'Down' keys.

### Audacity Version

current master / alpha / nightly build

### Operating system

macOS

### Additional context

This is also reproduced on latest public version.
This is not reproducible on Windows or Linux

**Comment by petersampsonaudacity** [comment]
Testing on W10 with latest branch build for New Export:  audacity-win-3.4.0-alpha-20230724+d3d98a4-x64

This appears to work properly on Windows

**Comment by petersampsonaudacity** [comment]
@DavidBailes @chinakov 

Testing on Macbook Pro Monterey 12.6.7 with @vsverchinsky 's latest branch build

This works fine for me on Mac

I can successfully navigate the Metadata table using the up/down and left/right arrows

So WORKS_FOR_ME on Mac

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
